src/Core/Migration/V6_6/Migration1738661307AddMediaIndices.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_6;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+class Migration1738661307AddMediaIndices extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1738661307;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $this->dropIndexIfExists($connection, 'media', 'idx.media.file_extension');
+        $connection->executeStatement(/** @lang MySQL */ 'CREATE INDEX `idx.media.file_extension` ON `media` (`file_extension`);');
```

Comment:
```suggestion
        $connection->executeStatement(<<<'SQL'
        CREATE INDEX `idx.media.file_extension` ON `media` (`file_extension`)
        SQL;);
```
I guess `@lang` is just an IDE-hint. I'd use the "SQL-string" (or whatever it is called), since I guess most IDEs will detext it as well :) 

Otherwise I'd comment above instead of inside. Don't think I have seen this inside in shopware before.

Reply:
It looked bad to use HEREDOC on such a short string, that's why I injected the language with an annotation instead of the HEREDOC stopword.

Reply:
Not much fan of putting it insided the brackets, but I'm fine with it.